nfafrontend/src/app/newproject/newproject.component.html

```diff
@@ -2,7 +2,7 @@ <h2>NFA-Projekt erfassen</h2>
 <div class ="div1">
   <form>
     <label>Kundenname</label>
-    <input type="text" class="txt" name="item" placeholder="Bundesagentur fuer Arbeit"style="float:right"><br><br>
+    <input type="text" class="txt" name="item" placeholder="Bundesagentur fuer Arbeit"style="float:right" [(ngModel)]="kundenName"><br><br>
```

Comment:
Even though it's a sample Text from msg, we should thing about a Translation, so all Team-Members can work on this the same way.